volunteers/models.py

```diff
@@ -2,54 +2,99 @@
+    emergency_phone = models.IntegerField(validators=[MinLengthValidator(
+        7, "Please enter a valid phone number (xxx-xxx-xxxx).")])
     notes = models.CharField(max_length=600)
     cross_cultural_experiences = models.CharField(max_length=600)
     why_FDPs = models.CharField(max_length=600)
     spirituality = models.CharField(max_length=600)
 
+
 class Interest(models.Model):
     volunteer = models.ForeignKey(Volunteer, on_delete=models.CASCADE)
     interest_name = models.CharField(max_length=50)
 
+
 class Language(models.Model):
     volunteer = models.ForeignKey(Volunteer, on_delete=models.CASCADE)
     language_name = models.CharField(max_length=30)
-    level_fluent = models.IntegerField(default=0)
+    level_fluent = models.CharField(
+        max_length=30,
+        choices=(
+            ("mother_tongue", "Native Speak"),
+            ("near_native", "Fluent"),
+            ("exclt_command", "Highly proficient in spoken and written"),
+            ('good_command', 'Good working knowledge'),
+            ('working_knowledge', 'basic communication skills')
+        )
+
+    )
+
 
 class Event(models.Model):
     volunteer = models.ForeignKey(Volunteer, on_delete=models.CASCADE)
-    event_name = models.CharField(max_length=100, validators=[MinLengthValidator(1, "Please enter a valid event name.")])
+    event_name = models.CharField(max_length=100, validators=[
+                                  MinLengthValidator(1, "Please enter a valid event name.")])
     event_date = models.DateTimeField('Event date')
     event_description = models.CharField(max_length=200)
 
+
 class OrganizationAffiliation(models.Model):
     volunteer = models.ForeignKey(Volunteer, on_delete=models.CASCADE)
-    organization_name = models.CharField(max_length=50, validators=[MinLengthValidator(1, "Please enter a valid organization name.")])
+    organization_name = models.CharField(max_length=50, validators=[
+                                         MinLengthValidator(1, "Please enter a valid organization name.")])
+
 
 class Training(models.Model):
     volunteers = models.ManyToManyField(Volunteer)
-    training_type = models.CharField(max_length=30, validators=[MinLengthValidator(1)])
-    training_url = models.CharField(max_length=1000, validators=[MinLengthValidator(1)])
+    training_type = models.CharField(
+        max_length=30, validators=[MinLengthValidator(1)])
+    training_url = models.CharField(
+        max_length=1000, validators=[MinLengthValidator(1)])
     training_complete = models.IntegerField(default=0)
 
+
 class Role(models.Model):
     trainings = models.ManyToManyField(Training)
     volunteer = models.ForeignKey(Volunteer, on_delete=models.CASCADE)
-    role_name = models.CharField(max_length=30, validators=[MinLengthValidator(1)])
\ No newline at end of file
+    role_name = models.CharField(max_length=30, validators=[
+                                 MinLengthValidator(1)])
```

Comment:
Small semantic thing - but could you add an extra line at the end of the file? :)

Reply:
I am lost here!

Reply:
Every python file should end with a blank line.